images/kops-server-ng/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:stretch-slim
+ARG KOPS_SERVER
+# TODO
+# RUN apt-get update && apt-get install --yes --reinstall lsb-base
```

Comment:
Todo?

Reply:
Copy-pasted from the original kops-server image Dockerfile in `images/kops-server/Dockerfile`. I can't speak to what needs to be done.